apps/test/solutions/maze/bee/bee_clouds.js

```diff
@@ -26,16 +26,6 @@ var levelDef = {
     [ 0, 0, 0, 0, 0, 0, 0, 0 ],
     [ 0, 0, 0, 0, 0, 0, 0, 0 ]
   ],
-  'rawDirt': [
-    [ 0, 0, 0, 0, 0, 1, 1, 1 ],
-    [ 0, 1, 1, 0, 0, 1, 1, 1 ],
-    [ 0, 0, 0, 0, 0, 1, 1, 1 ],
-    [ 0, 0, 0, 0, 0, 0, 0, 0 ],
-    [ 2, 1, '+1FC', '-1', 1, 0, 0, 0 ],
-    [ 0, 0, 0, 0, 0, 0, 0, 0 ],
-    [ 0, 0, 1, 1, 0, 0, 0, 0 ],
-    [ 0, 0, 1, 1, 0, 0, 0, 0 ]
-  ],
```

Comment:
I actually added rawDirt to the tests with my first pass of Quantum Bee, when I was trying to strictly move everything to using the new maps rather than providing for backwards compatibility.